actions_helper/commands/run_preflight.py

```diff
@@ -0,0 +1,65 @@
+import click
+from botocore.client import BaseClient
+
+from actions_helper.commands.create_task_definition import create_task_definition
+from actions_helper.commands.wait_for_task_stopped import wait_for_task_stopped
+from actions_helper.outputs import RunPreflightOutput
+from actions_helper.utils import set_error
+
+
+def run_preflight_container(
+    ecs_client: BaseClient,
+    cluster: str,
+    service: str,
+    application_id: str,
+    image_uri: str,
+    deployment_tag: str,
+) -> RunPreflightOutput:
+    click.echo("Creating preflight task definition...")
+    deployed_preflight_task_definition = create_task_definition(
+        ecs_client=ecs_client,
+        application_id=application_id,
+        deployment_tag=deployment_tag,
+        image_uri=image_uri,
+    )
+
+    network_config = ecs_client.describe_services(
+        cluster=cluster,
+        services=[service],
+    )["services"][0]["networkConfiguration"]["awsvpcConfiguration"]
+
+    click.echo("Running preflight task...")
+    task_arn = ecs_client.run_task(
+        cluster=cluster,
+        count=1,
+        launchType="FARGATE",
+        networkConfiguration={
+            "awsvpcConfiguration": {
+                "subnets": network_config["subnets"],
+                "securityGroups": network_config["securityGroups"],
+                "assignPublicIp": "DISABLED",
+            },
+        },
+        taskDefinition=deployed_preflight_task_definition.latest_task_definition_arn,
+    )["tasks"][0]["taskArn"]
+
+    wait_for_task_stopped(ecs_client=ecs_client, cluster=cluster, task=task_arn)
+
+    stopped_preflight_container = ecs_client.describe_tasks(
+        cluster=cluster,
+        tasks=[task_arn],
+    )["tasks"][0]["containers"][0]
```

Comment:
There should be only a single container, or?
```suggestion
    (stopped_preflight_container,) = ecs_client.describe_tasks(
        cluster=cluster,
        tasks=[task_arn],
    )["tasks"][0]["containers"]
```